src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
```

Comment:
The method 'doWork' in a class that extends 'Machine' should not return a string. Instead, it should print out an informative message indicating that the Bulldozer is working. Consider using System.out.println with a message specific to the Bulldozer class.